project/src/components/favorites-empty-screen/favorites-empty-screen.tsx

```diff
@@ -0,0 +1,52 @@
+function FavoritesEmptyScreen(): JSX.Element {
+  return (
+    <div className="page page--favorites-empty">
+      <header className="header">
+        <div className="container">
+          <div className="header__wrapper">
+            <div className="header__left">
+              <a className="header__logo-link" href="main.html">
+                <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
```

Comment:
для удобства, лучше всего писать вот так, когда много пропс в компонент закидываешь

```suggestion
                <img 
                  className="header__logo" 
                  src="img/logo.svg" 
                  alt="6 cities logo" 
                  width="81" 
                  height="41" 
                />
```